Detections/MultipleDataSources/GainCodeExecutionADFSviaWMI.yaml

```diff
@@ -72,6 +84,35 @@ query: |
   ) on TargetLogonId
   ),
   (
+  WindowsEvent
```

Comment:
Here again the original did not check for an Event Id and we may need to. I think it is also good as a best practice.